game/scripts/vscripts/abilities/oaa_glaives_of_wisdom.lua

```diff
@@ -155,25 +203,30 @@ function modifier_oaa_glaives_of_wisdom:OnAttackLanded(keys)
 
     local bonusDamage = parent:GetIntellect() * bonusDamagePct
 
-    local damageTable = {
-      victim = target,
-      attacker = parent,
-      damage = bonusDamage,
-      damage_type = ability:GetAbilityDamageType(),
-      ability = ability
-    }
+    local damageTable = {}
+    damageTable.victim = target
+    damageTable.attacker = parent
+    damageTable.damage = bonusDamage
+    damageTable.damage_type = ability:GetAbilityDamageType()
+    damageTable.ability = ability
```

Comment:
I really just have to ask. Why would you write the damageTable this way instead of what it was before?

Reply:
Because I change the damage type when target has spell immunity.

Reply:
That's not exactly necessary, though. Something like:

```
local table = {
    x = y,
}

if a >= b then
    table.x = z
end
```

works with zero issue.

Reply:
I didn't know that.